.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          persist-credentials: false
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version-file: .nvmrc
+          registry-url: 'https://registry.npmjs.org'
```

Comment:
Couldn't find an exact example of npm publishing plus incremental-release, so I tried to go based on those reverted changes in the action - @dlockhart does this look good?

Reply:
Yep looks correct!